site/source/user-manual/service-guides/lightning.rst

```diff
@@ -8,10 +8,163 @@ Lightning
+
+    This will work with either (or both!) lightning implementations.
+
+      .. figure:: /_static/images/lightning/rtl-in-marketplace1.png
+        :width: 60%
+
+#. Install it and click on **"Configure"** just like with CLN.
+
+   .. figure:: /_static/images/lightning/rtl-needs-config.png
+    :width: 60%
+
+#. It will default to LND. In this case we are using a CLN node instead, so we will change the default setting as shown:
+
+   .. figure:: /_static/images/lightning/rtl-config-lnd.png
+    :width: 60%
+
+   Change to Core Lightning (CLN) and hit OK
+
+   .. figure:: /_static/images/lightning/rtl-change-to-cln.png
+    :width: 60%
+
+   Hit **"Save"**
+
+   .. figure:: /_static/images/lightning/rtl-config-save.png
+    :width: 60%
+
+#. Now hit **"Start"**
+
+   .. figure:: /_static/images/lightning/rtl-start.png
+    :width: 60%
+
+#. With RTL started, click **"Properties"**
+
+   .. figure:: /_static/images/lightning/rtl-click-properties.png
+    :width: 60%
+
+#. Copy the automatically generated password
+
+   .. figure:: /_static/images/lightning/rtl-copy-pass.png
+    :width: 60%
+
+#. Head back to the RTL service and click **"Launch UI"**
+
+   .. figure:: /_static/images/lightning/rtl-launch-ui.png
+    :width: 60%
+
+   Enter the copied password and log in:
+
+   .. figure:: /_static/images/lightning/rtl-enter-pass.png
+    :width: 60%
+
+#. Once in RTL, click **"On-chain"** then click **"Generate Address"**
+
+   .. figure:: /_static/images/lightning/rtl-generate-address.png
+    :width: 60%
+
+#. Send money to the generated address to add funds to your lightning wallet:
+
+   .. figure:: /_static/images/lightning/rtl-address-generated.png
+    :width: 60%
+
+   .. note:: Please do not send money to the address pictured above as we will not receive it. If you are intent on sending us money please `head here <donate.start9.com>`_
+
+#. Once your sats confirm on-chain you'll see this:
+
+   .. figure:: /_static/images/lightning/rtl-on-chain.png
+    :width: 60%
+
+#. Now we must add a peer with which to make channels. In this example we will be opening a channel with Start9 so we will add Start9's node as a peer. Click **"Peers/Channels"**:
+
+   .. figure:: /_static/images/lightning/rtl-peers-channels.png
+    :width: 60%
+
+#. Click **"Peers"** then **"Add Peer"**:
+
+   .. figure:: /_static/images/lightning/rtl-peers-add-peer.png
+    :width: 60%
+
+#. Enter **025d28dc4c4f5ce4194c31c3109129cd741fafc1ff2f6ea53f97de2f58877b2295@64.225.19.231:9735** - these are the peer details for Start9's lightning node - and click **"Add Peer"**:
+
+   .. figure:: /_static/images/lightning/rtl-start9-node-info.png
+    :width: 60%
+
+#. Then you can enter an amount (the size of the channel), select Private Channel (unless you want a public channel - see below), and a Fee Rate (check a `block explorer <https://mempool.space>`_ for an idea of current necessary fees):
```

Comment:
Maybe a note that this is a very small demo channel and that you will want a channel large enough to cover purchases that you intend to make via this channel